css/modern.css

```diff
@@ -1755,9 +1759,124 @@ table#collection-view-contents tbody tr td {
     background-color: var(--highlight) !important;
 }
 
+<<<<<<< HEAD
+
+/* Quizes */
+
+#s-assessment-view-submission-form .selected.correct  span.option-span {
+    background-color: var(--active) !important;
+    color: var(--text) !important;
+}
+
+
+#s-assessment-view-submission-form .incorrect span.option-span.legacy-true-false-question, #s-assessment-view-submission-form .incorrect span.option-span.legacy-multiple-choice-question {
+    background-color: var(--highlight) !important;
+    color: var(--text) !important;
+}
+
+/* California Assesment */
+
+.ca-assessment-delivery-take-assessment {
+    background-color: var(--background-color) !important;
+}
+
+.lrn-assess > div {
+    background-color: var(--background-color) !important;
+}
+
+.lrn.lrn-assess .lrn-right-region .right-wrapper .menu-overlay {
+    background-color: var(--border-color) !important;
+    color: var(--text) !important;
+}
+
+
+.lrn .pagination>li>a, .lrn .pagination>li>span {
+    background-color: var(--border-color) !important;
+
+}
+
+.lrn .pagination>li>a, .lrn .pagination>li>span {
+    background-color: var(--border-color);
```

Comment:
Don't use border color! This isn't a variable we're using, especially not for backgrounds. Please only use `primary`, `secondary` and `accent` for backgrounds

Reply:
needed a white background